NBitcoin.Tests/AltcoinTests.cs

```diff
@@ -208,7 +246,18 @@ public void DoesRPCCapabilitiesWellAdvertised()
 					{
 						AddressType = AddressType.Bech32
 					});
-					// If this fail, rpc support segwit bug you said it does not
+					if (IsElements(node.Network))
+					{
+//						if (address is BitcoinBlindedAddress blindedAddress)
+//						{
+//							address =blindedAddress.UnblindedAddress;
+//						}
+						//the above works too but might as well test this elements rpc call too
+						var addrInfo = await rpc.GetAddressInfoAsync(address.ToString(), rpc.Network);
+						address = addrInfo.Unconfidential;
```

Comment:
1. Why is this needed?  The `address.ScriptPubKey` of the confidential should be the same than the non confidential (check with a test)
2. It should be possible to get the unconfidentila address without a call to RPC, if so can you test that `addrInfo.Unconfidential` match what NBItcoin is returning?

Reply:
You're right, removed this